lib/modules/manager/velaci/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ProgrammingLanguage } from '../../../constants';
+import { DockerDatasource } from '../../datasource/docker';
+import { extractPackageFile } from './extract';
+
+const language = ProgrammingLanguage.Docker;
```

Comment:
```suggestion
export const language = ProgrammingLanguage.Docker;
```
Can there be other types of dependencies iin pipeline config or are there any plans to add? then this should be removed

```suggestion
```

Reply:
probably not extending outside of docker anytime soon at least